src/Check/IniFile.php

```diff
@@ -22,7 +22,7 @@ class IniFile extends AbstractFileCheck
      */
     protected function validateFile($file)
     {
-        if (! is_array($ini = parse_ini_file($file)) || count($ini) < 1) {
+        if (! is_array($ini = @parse_ini_file($file)) || count($ini) < 1) {
```

Comment:
Could we use something like `set_error_handler()` + `restore_error_handler()` here, perhaps? :thinking: 

Using `@` always bites us back later, so just wondering

Reply:
I was indeed thinking about that and decided to skip it here for sake of brevity. But using the error handler might actually allow us to add the reported message to the error-message.

So will add